docs/provider/daemon/getting_started.md

```diff
@@ -0,0 +1,138 @@
+
+2. Move the binaries into a folder in your PATH (eg. `/usr/local/bin`) and rename the daemon as just `tdexd` and the CLI as `tdex`.
+
+3. Give them executable permissions. (eg. `chmod a+x /usr/local/bin/tdexd` and `chmod a+x /usr/local/bin/tdex`)
+
+
+Now you're ready to start the daemon:
+
+```sh
+# Run on Liquid network connecting to blockstream.info for sourcing blockchain data
+$ tdexd
+
+# Run on Liquid connecting to a local explorer
+$ export TDEX_EXPLORER_ENDPOINT="http://127.0.0.1:3001"
+$ tdexd
+
+# Run on Regtest connecting to a local explorer and using regtest LBTC asset hash.
+$ export TDEX_NETWORK="regtest"
+$ export TDEX_BASE_ASSET="5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225"
+$ export TDEX_EXPLORER_ENDPOINT="http://127.0.0.1:3001"
+$ tdexd
+
+# Run on Liquid and specify USDt as base asset instead of default L-BTC
+$ export TDEX_BASE_ASSET="ce091c998b83c78bb71a632313ba3760f1763d9cfcffae02258ffa9865a37bd2"
+$ tdexd
+```
+
+You will find the data directory at the path `~/.tdex-daemon` if using Linux or, instead, `~/Library/Application\ Support/Tdex-daemon` if using a MacOs.
+
+Next step is to [configure the operator CLI](#configure-operator-cli).
+
+<!-- Now you are ready to [deposit funds](setup_wallet.md) to create your first market and start accepting incoming trades. -->
+
+## Configure operator CLI
+
+Now that your daemon is up and running, you need to configure the `tdex` CLI to communicate with the Operator interface.
+
+Depending on how you started your daemon, you want to initialize your CLI like:
+
+```sh
+# By default it looks for the daemon operator gRPC interface on localhost:9000
+$ tdex config init
+# If the daemon is running on regtest
+$ tdex config init --network regtest --explorer_url http://localhost:3001
+# or on a remote machine
+$ tdex config init --rpcserver example.com:9000
+```
+
+You can always check the current state with the following command
+
+```sh
+$ tdex config
+# network: regtest
+# explorer_url: http://localhost:3001
+# rpcserver: localhost:9000
+# tls_cert_path:
+# no_macaroons: true
+# macaroons_path:
+```
+
+The entries of the state of the CLI are configurable with:
+
+```sh
+# To set the current network
+$ tdex config set <state_key> <value>
+```
+
+If, for example, you configured your daemon to authenticate/authorize access to Operator interafce RPCs with macaroons, you must set the path to the `admin.macaroon` and also the one for the TLS certificate `cert.pem` like:
+
+```sh
+# To set the path where to find admin.macaroon
+$ tdex config set macaroons_path ~/.tdex-daemon/macaroons
+# To set the path where to fing cert.pem
+$ tdex config set tls_cert_path ~/.tdex-daemon/tls
+```
+
+If instead, your dameon's Operator interface is not proteced by this type of authentication/authorization, then you need to run:
+
+```sh
+# To not use macaroons auth
+$ tdex config set no_macaroons true
```

Comment:
What happens if the ENV VAR of the daemon has macaroons activated and here we do no_macarrons or viceversa? there could inconsistencies

Reply:
If the daemon is configured to not use macaroons auth, then no macaroons file will be created so it won't be possible to configure the CLI to use some macaroons.
In another scenario, where the daemon has been started the first time with mac auth, then it was stopped and restarted with no mac auth, if the CLI is still using some of the macaroons that are present in the daemon datadir, then every command will return the same error:
```
[tdex] rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: EOF"
```
This because the client (CLI) tries to establish an encrypted connection with server (daemon)  that doesn't have TLS termination enabled on its Operator interface.

In the opposite way, if the daemon has macaroons auth enabled and the CLI has `no_macaroons` set to true, then the latter won't be usable essentially.
The reason is that when the Operator interface has macaroons auth enabled, its dialer is configured with a TLS connection level security credential. This makes every command of the CLI not usable because this doesn't make use of any TLS certificate in "`no_macaroons` mode".
Every command will return the same error:
```
[tdex] rpc error: code = Unavailable desc = connection closed
```